internal/pkg/cli/svc_init.go

```diff
@@ -316,46 +317,53 @@ func (o *initSvcOpts) askSvcName() error {
 	return nil
 }
 
-// askDockerfile prompts for the Dockerfile by looking at sub-directories with a Dockerfile.
-func (o *initSvcOpts) askDockerfile() error {
-	if o.DockerfilePath != "" {
-		return nil
-	}
-
-	dockerfiles, err := listDockerfiles(o.fs, ".")
+func askDockerfile(wkldName string, fs afero.Fs, p prompter) (string, error) {
```

Comment:
i feel like instead of making this a private function, we could potentially move the logic to `selector` pkg as we did for [DeployedService](https://github.com/aws/copilot-cli/blob/bd820736f5d740cc6a05b304fc90289e91cb14ae/internal/pkg/term/selector/selector.go#L140). One benefit for this is we could make our CLI pkg clearer to have a better abstraction.